contract-ui/tabs/overview/components/PermissionsTable.tsx

```diff
@@ -50,7 +50,7 @@ export const PermissionsTable: React.FC<PermissionsTableProps> = ({
         },
         [] as { member: string; roles: string[] }[],
       ) || []
-    );
+    ).filter((m) => m.member !== "0x0000000000000000000000000000000000000000");
```

Comment:
```suggestion
    ).filter((m) => m.member !== "0x0000000000000000000000000000000000000000");
```
dont hardcode this, get it from ethers, `constants.AddressZero`